src/WebJobs.Script.WebHost/Diagnostics/MetricsEventManager.cs

```diff
@@ -315,6 +291,35 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+        // TODO: When StyleCop is updated to a release after Dec 2021, this can be a much simpler readonly record struct
```

Comment:
Is this fixable now?

Reply:
So yes and no - there has been a package update but there's a case sensitivity question in play here. If we can ignore case sensitivity, this can be faster (as can other areas - we take some papercuts on string comparisons in various places since that's a less efficient path but one commonly hit). I think event names and functions are likely stable but I'm not sure. Will update the comment with the further findings here /cc @mathewc for thoughts on casing.